src/main/java/duckie/task/TaskList.java

```diff
@@ -0,0 +1,100 @@
+package duckie.task;
+
+import java.util.ArrayList;
+
+import duckie.exception.*;
+import duckie.Ui;
+
+/**
+ * Contains all the saved tasks in a List form
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    /**
+     * Instantiate an empty list
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    /**
+     * Instantiate a TaskList with pre-saved tasks
+     * @param tasks ArrayList containing pre-saved tasks
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * Return the ArrayList containing the current tasks
+     * @return ArrayList containing current tasks
+     */
+    public ArrayList<Task> getTaskList() {
+        return tasks;
+    }
+
+    /**
+     * Return the actual Index of a task in the ArrayList
```

Comment:
Should be "Returns..." as "In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.)"?